src/cmd/list.cpp

```diff
@@ -25,7 +25,7 @@ namespace cmd {
 			("i,input", "File(s) to list", cxxopts::value<std::vector<std::string>>())
 			("o,output", "Output file to write results to", cxxopts::value<std::string>()->default_value("stdout"))
 			("d,directory", "Working directory", cxxopts::value<std::string>()->default_value("."))
-			("m,maxdepth", "Max recursion depth", cxxopts::value<int>()->default_value("0"))
+			("m,maxdepth", "Max recursion depth", cxxopts::value<int>()->default_value("-1"))
```

Comment:
Keep this consistent with `ddb info`:

```
-d, --depth arg     Max recursion depth (default: 0)
```

Reply:
Ah, nvm, we have `-d` for `--directory`.

Reply:
The funny thing is that if you have two options with the same shortcut it will crash without saying anything.